### PR TITLE
Task 31

### PR DESCRIPTION
**Pull request by j-ae**
[Task 31](https://github.com/Appers-lab/billihomes-frontend/issues/41)

1. Changed the task10 Icons to the given Icons. For this, I used inline-svg tag.

2. Changed the task 10 background to the given image.

3. Designed a mobile version property selection section.

### CODE REVIEW

File: tasks/task-031 (enhancing prop selection page)/mobile-prop-selection.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+    <article class="s-panel text-center">
+
+        <div class="ss-prop-selection-section">
+
+            <p class="ss-prop-selection-section__q mb-2">What type of property you are looking for?</p>
+
+            <div class="row mb-2">
+
+                <div class="col-6 d-flex flex-column align-items-center">
```
Comment (pmvald) [inline tasks/task-031 (enhancing prop selection page)/mobile-prop-selection.html:18]: Do not make col- blocks flex (or apply any other layout style to them). Instead create a `<div>` inside the col- block and apply all the layout styles to it (why? Hint: col- blocks fall into the negative margin of the row, and they are also touching each other in row, but a containing `<div>` will solve all these problems ...)

File: tasks/task-031 (enhancing prop selection page)/mobile-prop-selection.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                    <!-- html code for a single icon circle -->
+                    <article class="s-icircle">
+
+                        <!-- icon section -->
+                        <section class="s-icircle__icon">
+                            <svg preserveAspectRatio="xMidYMid meet" viewbox="0 0 93.3333 93.3333" xmlns="http://www.w3.org/2000/svg"><path d="M80 70V0H40.976L16.6667 24.3093V70H20V25.6907L42.3573 3.3333h10.976V70h3.3334V3.3333h20V70zM46.6667 20H40v3.3333h6.6667zM70 10h-6.6667v3.3333H70zm0 10h-6.6667v3.3333H70zm0 10h-6.6667v3.3333H70zm0 10h-6.6667v3.3333H70zm0 10h-6.6667v3.3333H70zm-6.6667 13.3333H70V60h-6.6667zm-23.3333 0h6.6667V60H40zm-13.3333 0h6.6666V60h-6.6666zm20-13.3333H40v3.3333h6.6667zm-13.3334 0h-6.6666v3.3333h6.6666zm13.3334-10H40v3.3333h6.6667zm-13.3334 0h-6.6666v3.3333h6.6666zm13.3334-10H40v3.3333h6.6667zm-13.3334 0h-6.6666v3.3333h6.6666zm13.3334 50h-3.3334v6.6667h3.3334zM30 80h-3.3333v6.6667H30zM0 93.3333h93.3333V90H0zm83.3333-20v3.3334H80v10h-3.3333v-10h-20v10h-3.3334v-10H20v10h-3.3333v-10h-3.3334v-3.3334z" fill="white"/></svg>
```
Comment (pmvald) [inline tasks/task-031 (enhancing prop selection page)/mobile-prop-selection.html:24]: Good!

File: -sass/styles/_icircles.scss [hunk 3]
```diff
@@ -1,49 +1,43 @@
 
 .s-icircle {
-    margin-left: 50px;
-    width: 200px;
-    height: 200px;
+    --circle-radius: 12.5rem;
+    --circle-color: #{$primary-color};
+    --text-color: white;
+    --box-shadow: 0 1rem 3rem rgba(0, 0, 0, 0.3);
```
Comment (pmvald) [inline -sass/styles/_icircles.scss:6]: It's a good thing to use css varibles, but only when there is a reason for it. In other words, we use the css variables when there is (or may be) an occasion that the value is going to be changed; like when we have a BEM modifier, or some external stylesheet may use the css variable to change the component ... but if a value is going to be the same and constant cross the stylesheet then we may just hard code it or use scss variable (scss variables serve as constant variables, kind of like `const` keyword in C++ or `final` in Java). 

However, in this case of `s-icircle` I see some occasion for css variables. I see you made two versions of s-icircle class: one for desktop and one for mobile. But can we unify these two versions into one code and switch between them using some modifier? How mobile version is different from teh desktop version? Is it just smaller font-size? smaller border-radius? In that case can we just a modifier like `--sm`? It is up to you to decide. If they are not very different, then just define modifiers and change some css variables to make it mobile (or smaller?) version. Also, it is a good idea to define theme modifier `--light` and `--dark` for color and background color (right now it is dark theme - in the light theme the background becomes white and text color becomes primary color).

File: tasks/task-031 (enhancing prop selection page)/prop-selection-page.html [hunk 4]
```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<head>
+    <link rel="stylesheet" href="../../css/bundle.css">
+</head>
+
+<body style="background-image: url('./wallpaper2.png'); background-position-y: 100px;">
```
Comment (pmvald) [inline tasks/task-031 (enhancing prop selection page)/prop-selection-page.html:6]: An aesthetic idea: can we make the background picture more fade (like opacity 0.3). why? (hint: UI/UX design principal! not to draw the user attention to the background image by fading it ...)

File: tasks/task-031 (enhancing prop selection page)/mobile-prop-selection.html [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+        <div class="ss-prop-selection-section">
+
+            <p class="ss-prop-selection-section__q mb-2">What type of property you are looking for?</p>
+
+            <div class="row mb-2">
+
+                <div class="col-6 d-flex flex-column align-items-center">
+                    <!-- html code for a single icon circle -->
+                    <article class="s-icircle">
```
Comment (pmvald) [inline tasks/task-031 (enhancing prop selection page)/mobile-prop-selection.html:20]: For iPhone 5/SE the circles overlapped (use developer tools to vie won this device) - could be due to the issue mentioned in the previous comment

**Review by pmvald — CHANGES_REQUESTED** [review]

**Comment by j-ae** [comment]
@pmvald 

1. Fixed "Do not make col- blocks flex".
2. Deleted _icircle.m.sass and added a modifier "--sm" for mobile version.
3. Added opacity for the background.

**Review by pmvald — APPROVED** [review]

**Comment by pmvald** [comment]
@j-ae cool!

**Pull request merged**